app/views/schemes/details.html.erb

```diff
@@ -61,13 +40,32 @@
         :name,
         legend: { text: "What is this type of scheme?", size: "m" } %>
 
-      <% care_acts_selection = Scheme.registered_under_care_acts.keys.reverse.map { |key, _| OpenStruct.new(id: key, name: key.to_s.humanize) } %>
-
       <%= f.govuk_collection_radio_buttons :registered_under_care_act,
-        care_acts_selection,
-        :id,
-        :name,
-        legend: { text: "Is this scheme registered under the Care Standards Act 2000?", size: "m" } %>
+                                           @scheme.care_acts_options_with_hints,
+                                           :id,
+                                           :name,
+                                           :description,
+                                           legend: { text: "Is this scheme registered under the Care Standards Act 2000?", size: "m" },
+                                           bold_labels: false %>
+
+      <% organisations = Organisation.all.map { |org| OpenStruct.new(id: org.id, name: org.name) } %>
```

Comment:
Have we dropped the null option here? Should we have?

Reply:
This is not an optional field and this page is never accessed without existing form ( e.g. you can't get there without populating this field in the first place ).